packages/cli/package.json

```diff
@@ -22,6 +22,7 @@
   "publishConfig": {
     "access": "public"
   },
+  "engines" : { "node" : ">=0.12" },
```

Comment:
will this trigger a warning on `yarn install` ?

Reply:
Yes

Reply:
It will block the whole install even. Only with `--ignore-engines` this will go through